browser/resources/settings/brave_overrides/settings_menu.js

```diff
@@ -99,10 +100,27 @@ RegisterStyleOverride(
         height: 20px;
       }
 
+      a[href] { position: relative; }
+      a[href].iron-selected::before {
+        content: "";
+        position: absolute;
+        top: 50%;
+        left: -30px;
```

Comment:
@petemill, is it safe for this to be a hardcoded value? Is there a CSS variable for the padding on the settings menu that should be used instead?

Reply:
Well, looks like the 30px padding comes from higher up
https://github.com/brave/brave-core/blob/c555d0f231d7b2785608305574cc4b07821e03cb/browser/resources/settings/brave_overrides/settings_menu.js#L57

And yes it does look like we have a variable - `var(--brave-settings-menu-padding)` can be used instead!

I just wish we had one for the height as that hard-coding isn't great, or I'm wondering if we could use 100% or flex?

Reply:
100% won't work since the design indicates that the accent line is taller than the menu item itself. That said, we could determine a ratio/percentage of the menu item height.

Reply:
```suggestion
        left: calc(-1 * var(--brave-settings-menu-padding));
```

@spylogsster, can you attempt the fix above locally to ensure it works properly?